src/main/resources/db/changelog/changes/schema.xml

```diff
@@ -713,4 +713,25 @@
         <addPrimaryKey tableName="test_question" columnNames="test_id, question_id"/>
     </changeSet>
 
+    <changeSet id="add_coach_answer_table" author="SemyonKorostik">
+        <createTable tableName="coach_answer">
+            <column name="question_id" type="int" >
+                <constraints nullable="false"/>
+            </column>
+            <column name="test_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="comment" type="text"/>
+        </createTable>
```

Comment:
You need to add primary key to this table like this:
`<addPrimaryKey tableName="coach_answer" columnNames="test_id, question_id"/>`